spec/lib/rbac/access_spec.rb

```diff
@@ -5,14 +5,15 @@
   let(:rbac_access) { described_class.new(resource, verb) }
   let(:api_instance) { double }
   let(:rs_class) { class_double("RBAC::Service").as_stubbed_const(:transfer_nested_constants => true) }
+  let(:opts) { { :limit => RBAC::Access::DEFAULT_LIMIT }}
 
   before do
     allow(rs_class).to receive(:call).with(RBACApiClient::AccessApi).and_yield(api_instance)
   end
 
   it "fetches the array of plans" do
     with_modified_env :APP_NAME => app_name do
-      allow(RBAC::Service).to receive(:paginate).with(api_instance, :get_principal_access, {}, app_name).and_return([access1])
+      allow(RBAC::Service).to receive(:paginate).with(api_instance, :get_principal_access, opts, app_name).and_return([access1])
```

Comment:
@mkanoor not sure I understand why we need to pass in `opts` instead of `{}` for the specs outside of matching the arguments used in the code.

Reply:
@syncrou The arguments have to match, when we call the access functions its going to pass in the limit of 500 previously we were using the default and hence we had {}